schemaregistry/schemaregistry.md

```diff
@@ -143,33 +146,102 @@ Conceptually, a schema is a description of a data structure. Since data
 structures evolve over time, the schema describing them will also evolve over
 time. Therefore, a schema often has multiple versions.
 
-For simple scenarios, the API allows for version management to be automatic and
-transparent. Whenever a schema is updated, a new version number is assigned and
-prior schema versions are retained. This specification does not mandate a
-retention policy, but implementations MAY retire and remove outdated schema
-versions.
-
-The latest available schema is always the default version that is retrieved when
-the URL to the schema is given without the version specified.
+In this specification, the ["schema version"](#225-schema-version-attributes)
+refers to an individual schema document. The ["schema"](#224-schema-attributes)
+is a management bracket for those documents that helps enforcing consistency
+across versions, including compatibility policies.
 
 A newer schema version might introduce breaking changes or it might only
 introduce careful changes that preserve compatibility. These strategies are not
 subject of this specification, but the API provides a conflict handling
 mechanism that allows an implementation to reject updates that do not comply
 with a compatibility policy, if one has been implemented.
 
-### 2.2.1. Multi-format schema groups
+For simple scenarios, the API allows for version management to be automatic and
+transparent. Whenever a schema is updated, a new version number is assigned and
```

Comment:
Do you mean for this to be normative? I wondering if it should be "a new version number MUST be assigned"  ?

Reply:
did you want to update this?